README.md

```diff
@@ -71,7 +71,7 @@ export default class App extends Component {
     <td>value</td>
     <td>string / number</td>
     <td><strong>true</strong></td>
-    <td>''</td>
+    <td></td>
```

Comment:
@apollonian is this good?

Reply:
Yep! separator is not an empty string by default so this change looks good.

Reply:
Cool then.

Reply:
@Rishabh510 can you keep it same?

Reply:
Sure